app/views/recipes/index.html.erb

```diff
@@ -1,14 +1,15 @@
-<p style="color: green"><%= notice %></p>
+<%= link_to "<-- Back to user profile", user_url(current_user) %>
+<%= link_to "New recipe", new_user_recipe_path %>
 
 <h1>Recipes</h1>
 
-<div id="recipes">
-  <% @recipes.each do |recipe| %>
-    <%= render recipe %>
-    <p>
-      <%= link_to "Show this recipe", recipe %>
-    </p>
-  <% end %>
-</div>
+<% if @recipes.empty? %>
+  <p>Your recipes list is empty.</p>
+<% else %>
+  <div id="recipes">
+    <% @recipes.each do |recipe| %>
+      <%= render recipe %>
+    <% end %>
```

Comment:
[OPTIONAL] This page is currently displaying the following bullet warning:
![image](https://user-images.githubusercontent.com/42799579/193890677-32328f7f-fa44-40ac-90aa-a47130c583de.png)

Please consider changing @recipes.each to @recipes.includes(:user).each to avoid N+1 problems 😀.

Reply:
@Gopxfs Thanks for noting the N + 1 problem. Now it is solved. 😁